support-workers/cloud-formation/src/templates/cfn-template.yaml

```diff
@@ -343,7 +343,7 @@ Resources:
                 - Name: PaymentProvider
                   Value: Stripe
                 - Name: ProductType
-                  Value: DigitalPack
+                  Value: DigitalSubscription
```

Comment:
I think the product type is still coming through as DigitalPack, so the alarm is going off but the data is still coming through.

Reply:
https://github.com/guardian/support-frontend/pull/2899